src/Proxy/FilterPathProxy.php

```diff
@@ -17,7 +16,7 @@
 use ValueObjects\Web\Domain;
 use ValueObjects\Web\PortNumber;
 
-class CalendarSummaryProxy extends Proxy
+class FilterPathProxy extends Proxy
```

Comment:
Hardcoden op GET is volgens mij een zinnige default of zou je eerder meer http-methodes toelaten? Als er later toch behoefte aan is, kan je een `FilterPathMethodProxy` voorzien waar je wel de methodes kan bepalen?